tests/search/test-class-search.php

```diff
@@ -127,7 +127,7 @@ public function test__vip_search_connection_constants() {
 		$es = new \Automattic\VIP\Search\Search();
 		$es->init();
 
-		$this->assertEquals( EP_HOST, 'https://es-endpoint1' );
+		$this->assertEquals( true, 'https://es-endpoint1' === EP_HOST or 'https://es-endpoint2' === EP_HOST );
```

Comment:
How about using `assertContains()` here?
```suggestion
		$this->assertContains( EP_HOST, VIP_ELASTICSEARCH_ENDPOINTS );
```